transcom/mymove
# [ MB-16350, MB-16395 ] Adding DOSFSC and DDSFSC test data

## Jira tickets

- [MB-16350](https://dp3.atlassian.net/browse/MB-16350)
- [MB-16395](https://dp3.atlassian.net/browse/MB-16395)

## Summary

- [x] Adding DOSFSC to the MTO Service Item Factory
- [x] Adding DDSFSC to the MTO Service Item Factory

## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/run-storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/about/application-setup/milmove-local-client/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/about/development/testing)

### How to test

1. Make sure the tests pass and the Go code is compiling

This PR can't be tested well as it's mostly setup for the linked Jira tickets

[MB-16350]: https://dp3.atlassian.net/browse/MB-16350?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[MB-16395]: https://dp3.atlassian.net/browse/MB-16395?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
